packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/testHelpers/testHelpers.js

```diff
@@ -0,0 +1,26 @@
+import InclusiveLanguageAssessment
+	from "../../../../../src/scoring/assessments/inclusiveLanguage/InclusiveLanguageAssessment";
+import Paper from "../../../../../src/values/Paper";
+import Factory from "../../../../specHelpers/factory";
+
+/**
+ * Tests multiple identifiers of one category. For example a singular noun and its plural forms.
+ *
+ * @param {Array} assessments The array of assessments to use.
+ * @param {Array} texts    The array of texts to check for each entry.
+ * @param {Array} identifiers The array of identifiers.
+ * @param {Array} feedbacks The array of the feedbacks for each non-inclusive phrase.
+ * @param {Number} score    The expected assessment score for this category.
+ * @returns {void}
+ */
+export function testMultipleForms( assessments, texts, identifiers, feedbacks, score ) {
+	identifiers.forEach( ( identifier, i ) => {
```

Comment:
I love this idea 😍 But I would really consider to not depend on the **order** of elements in the arrays you pass to this function. You could instead send individual checks (and rename the function to `testInclusiveLanguageAssessment` or something similar) OR send an array objects into this function, e.g.

```
[ 
  { identifier: "some identifier", text: "some text", expectedFeedback: "some feedback", expectedScore: "someScore" }, 
  { identifier: "some identifier", text: "some text", expectedFeedback: "some feedback", expectedScore: "someScore" } 
]
```

And for `assessments` you could consider fetching all inclusive language assessments instead of passing it as a parameter.

Also, because this file does not contain any tests itself, running `yarn test incluseLanguage` fails: "Your test suite must contain at least one test."

Reply:
This is a really good point! I was also a bit annoyed by the fact that it depends on the order of elements. Thank you for your suggestion! I'll adapt it that way :D